pkg/network/usm/postgres_monitor_test.go

```diff
@@ -59,18 +59,32 @@ func generateSelectLimitQuery(limit int) string {
 	return fmt.Sprintf("SELECT * FROM dummy limit %d", limit)
 }
 
+// testContext shares the context of a given test.
+// It contains common variable used by all tests, and allows extending the context dynamically by setting more
+// attributes to the `extras` map.
+type pgTestContext struct {
+	// The address of the server to listen on.
+	serverAddress string
+	// The port to listen on.
+	serverPort string
```

Comment:
nit: can be changed to postgresPort as being used in the tests

Reply:
I think it should be clear enough that's postgres (we're in postgres_monitor_test, and the struct is `pgTest...`